ftw/upgrade/workflow.py

```diff
@@ -126,18 +126,23 @@ def _migrate_workflow_history(self, context, old_wf, new_wf):
         if wfhistory is None or old_wf not in wfhistory:
             return
 
-        def _migrate_action(entry):
+        def _migrate_transition(entry):
```

Comment:
@jone I think there is some disparity here, since the actions don't get migrated if they aren't in the transistion mapping, so at the moment we actually only migrate transitions. Things like "file downloaded" would be dropped.